topics/ecology/tutorials/MetaShARK_tutorial/tutorial.md

```diff
@@ -0,0 +1,309 @@
+
+<img src="./Images/7_MetaShARK_spatialinfo.png" width=100%>
+
+<p align="justify">Next step is devoted to specifying geographic coverage.<br><br> You can use a method between "columns" or "custom". "Custom" allows you to create one to several geographical sites using a map widget where you can draw limits of each site or enter directly latitude and longitude coordinates. "Columns" method, used here, allows you to specify an attribute containing site names then associated latitude and longitudes attributes.</p>
+
+<img src="./Images/8_MetaShARK_geocov.png" width=100%>
+
+<p align="justify">Now geographic coverage is set, one can specific taxonomic coverage.<br><br> To do so, you can select a data attribute containing taxonomic information then select kind of notation you want to have and finally on which taxonomic authority (or authorities) information will be compared. Note that this can take a while if you have a lot of taxons and time is duplicated for each selected additional authority.</p>
+
+<img src="./Images/9_MetaShARK_taxcov.png" width=100%>
+
+<p align="justify">Now we can fill personal informations.<br><br> To do so, the easiest way is to provide ORCID identifiers for each individual person involved as creator, contact and/or PI. Depending on the information filled in ORCID by each individual and on the level of accessibility of each, all field can be automatically filled. If "PI" is selected, you can specify a project name, funder name and related funding number.</p>
+
+<img src="./Images/10_MetaShARK_personal.png" width=100%>
+
+Then you can add final elements as:
+
+- An abstract (writing directly on the dedicated field or uploading a text file containing the abstract)
+
+<img src="./Images/11_MetaShARK_abstract.png" width=100%>
+
+- Methods (writing directly on the dedicated field or uploading a text file, can be in markdown, containing details of the methods used to create data files)
+
+<img src="./Images/12_MetaShARK_methods.png" width=100%>
+
+- Keywords, who can be linked to keyword thesaurus. This allows you to create "group" of keywords and/or refer to existing terms classifications as we can find in terminological resources such as ontologies or thesaurus.
+
+<img src="./Images/13_MetaShARK_keywords.png" width=100%>
+
+> [!TIP]
+> **You can add [semantic annotations](https://eml.ecoinformatics.org/semantic-annotation-primer)!**
+ 
+<p align="justify"> To do so, you need to reach the MetaShARK parameters (upper right icon) then enter your CEDAR token. To create a CEDAR account, you can register here http://cedar.metadatacenter.org/.</p>
+
+ API key format to enter is something like: 
+```
+api 205b1e521f2eaf0ad4a361c438b63205b1e521f2eaf0ad4a361c438b63c438b63
```

Comment:
i checked that it isn't a live api key :) and in the process found this could use more documentation.

1. you need to register
2. and then go to the 'profile' on cedar.metadatacenter.org
3. and there you can find the API key (I was having issues locating it, otherwise.)

Reply:
Done, thank you!